gramps/gui/widgets/styledtexteditor.py

```diff
@@ -319,7 +319,8 @@ def make_tooltip_from_link(self, link_tag):
             if url.startswith("gramps://"):
                 obj_class, prop, value = url[9:].split("/")
                 display = simple_access.display(obj_class, prop, value) or url
-        return display
+        return display + \
+            (_("\nCtrl-Click to follow link") if self.get_editable() else '')
```

Comment:
This may be Command-click on a Mac.  I see that @jralls has opened PR #474 to address similar issues.

Reply:
I was hoping he would support either control-click or Command-click; that way anyone who was used to the way Gramps is now could continue to do the same.  And I would not have to add an 'if mac()'...

Reply:
PR #474 does indeed make it "command-click" on a Mac. The control-click behavior never worked on Macs because is_right_click() processed the event first and opened the context menu.